src/Codemirror.js

```diff
@@ -1,26 +1,28 @@
 const React = require('react');
 const ReactDOM = require('react-dom');
 const findDOMNode = ReactDOM.findDOMNode;
```

Comment:
Maybe 😄  Seems out of scope of this PR though. Probably worth considering in a follow-up!